### ISSUE TITLE
This is not an issue

### ISSUE THREAD

**Issue by sashaiw** [jpope15/STL_Reimplementations#1]
I just wanted to let you know that from now on I will be using this instead of the standard library whenever possible.

**Comment by bryanmontalvan** [comment]
I agree, I think this reimplementation of a library is a great move for C++69

I'm currently working with my team to use this library instead of the STL.

Great work @jpope15 !
